python-lib/design_experiment/helpers.py

```diff
@@ -0,0 +1,15 @@
+from io import StringIO
+from dataiku.core.dkujson import dumps
```

Comment:
Small remark: your library should not depend on dataiku function. It should be self contained and can be used outside of dss